src/main/java/BotManager.java

```diff
@@ -0,0 +1,150 @@
+import java.util.ArrayList;
+
+public class BotManager {
+    private final String name;
+    private ArrayList<Task> tasks;
+
+    BotManager(String name) {
+        this.name = name;
+        this.tasks = new ArrayList<Task>();
+    }
+
+    void greeting() {
+        System.out.println("    Hello! I'm " + name);
+        System.out.println("    What can I do for you?\n");
+    }
+
+    void exit() {
+        System.out.println("    Bye. Hope to see you again soon!\n");
+    }
+
+    private Task createTask(String prompt) throws DukeException {
+        String[] order = prompt.split(" ", 2);
+        String taskType = order[0];
+        switch (taskType) {
+        case "todo":
```

Comment:
Like that you adhered to the coding standard for something like switch case statements, a lot of my peers overlooked this.